apps/api/src/app/user/update-user-setting.dto.ts

```diff
@@ -30,7 +30,7 @@ export class UpdateUserSettingDto {
   @IsOptional()
   colorScheme?: ColorScheme;
 
-  @IsIn(<DateRange[]>['1d', '1y', '5y', 'max', 'ytd'])
+  @IsIn(<DateRange[]>['1d', '1y', '5y', 'max', 'ytd', 'mtd', 'wtd'])
```

Comment:
Can you please order alphabetically?
`max`, `mtd`, `wtd`, `ytd`

Reply:
sure thing